src/root/rootobject.d

```diff
@@ -37,6 +38,7 @@ alias DYNCAST_IDENTIFIER = DYNCAST.identifier;
 alias DYNCAST_TUPLE = DYNCAST.tuple;
 alias DYNCAST_PARAMETER = DYNCAST.parameter;
 alias DYNCAST_STATEMENT = DYNCAST.statement;
+alias DYNCAST_CONDITION = DYNCAST.condition;
```

Comment:
I will address the removal of all this `alias`-es in a separate PR.
@WalterBright